core/variant/variant_utility.cpp

```diff
@@ -933,12 +933,7 @@ String VariantUtilityFunctions::str(const Variant **p_args, int p_arg_count, Cal
 	String s;
 	for (int i = 0; i < p_arg_count; i++) {
 		String os = p_args[i]->operator String();
-
-		if (i == 0) {
-			s = os;
-		} else {
-			s += os;
-		}
+		s += os;
```

Comment:
Why these unrelated changes? Please restore, if they should be done they should be done in a separate PR

Reply:
Done.